.github/workflows/python-cicd-units.yaml

```
@@ -0,0 +1,21 @@
+name: Python CICD pipelines - Unittesting
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install  -r requirements.txt
```

After looking at it a bit more I think there's a better way than my previous suggestion.

I'm dropping the suggestion in line here, based on my comment on the [specific commit](https://github.com/GeoscienceAustralia/gnssanalysis/commit/a8bcf609832303fb2f66ebc8bf139950aaef8852#r144575145)
```suggestion
          sed -i "/{{VERSION_PLACEHOLDER}}/d" setup.py
          pip install .
```

---

Having the versioner in place, if there is a tag, the version is the tag, else the commit hash. 
We change it in the setup.py so we can install manually some to make some tests.

---

Oops, this is in the housekeeping branch